scripts/vagrant/multi/start_vagrant.sh

```diff
@@ -0,0 +1,134 @@
+#!/bin/bash
+
+set -xe
+
+export BOX="ubuntu/xenial64"
+PROVIDER=${PROVIDER:-virtualbox}
+
+if [[ "$PROVIDER" != "virtualbox" ]]; then
+    if [[ "$USER" == "" ]]; then
+        echo "USER env var not set"
+        exit 1
+    fi
+    if [[ "$SSH_KEY" == "" ]]; then
+        echo "SSH_KEY env var not set"
+        exit 1
+    fi
+fi
+
+if [[ "$PROVIDER" == "google" ]]; then
+    export BOX="google/gce"
+
+    if [[ "$GOOGLE_PROJECT_ID" == "" ]]; then
+        echo "GOOGLE_PROJECT_ID env var not set"
+        exit 1
+    fi
+    if [[ "$GOOGLE_JSON_KEY_LOCATION" == "" ]]; then
+        echo "GOOGLE_JSON_KEY_LOCATION env var not set"
+        exit 1
+    fi
+
+    # Create ingress rules if not already exists.
+    MAYBE_M3COORDINATOR_INGRESS=$(gcloud --project=studious-saga-237001 compute firewall-rules list 2> /dev/null | tail -n +2 | awk '{ print $1 };' | grep default-allow-m3coordinator) || true
```

Comment:
More of a question, what is `studious-saga-237001`?

Reply:
It's the "Hosted Project" gcloud project, it seems to be the only one not attributed to a chronosphere environment.

Reply:
Is it ok to expose this?

Reply:
Yea, needed to expose this for traffic to be allowed from the node running the benchmarker and the coordinators. It's for testing anyways.